public/index.html

```diff
@@ -10,6 +10,7 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link href="https://fonts.googleapis.com/css2?family=Amiri&display=swap" rel="stylesheet">
```

Comment:
Why not Montserrat like you can see inside designs? Amiri has special style and font weight 400, 700 only

Reply:
Sure, it's just for style used by user, if he want to use own (so just used in example). I've left that for this example, but also added Monterserrat and set it on body to have it default (in index.scss)